src/test/ui/async-await/in-trait/async-associated-types.rs

```diff
@@ -0,0 +1,24 @@
+// edition: 2021
+
+#![feature(async_fn_in_trait)]
+#![allow(incomplete_features)]
+
+use std::fmt::Debug;
+
+trait MyTrait<'a, 'b, T> where Self: 'a, T: Debug + Sized + 'b {
+    type MyAssoc;// = (&'a T, &'b U);
```

Comment:
```suggestion
    type MyAssoc;
```

What's this comment about?

Reply:
I think it was in the hackmd snippet, I'll just delete it